tests/unit/Framework/AssertTest.php

```diff
@@ -1524,6 +1524,19 @@ public function testAssertCountThrowsExceptionIfElementIsNotCountable(): void
         $this->fail();
     }
 
```

Comment:
Are you already familiar with the @testdox annotation? It allows you to format the output. Normally the TestDox output formatter will break up CamelCase like in "InvalidArgumentException".

Example in this case:
/** @testdox InvalidArgumentException uses correct article for variable type */